src/config.js

```diff
@@ -4,8 +4,13 @@ const config = {
   pluginMap,
   languages: ['fi', 'sv', 'en'],
   activeLanguage: 'fi',
+  publicUrl: (typeof window !== 'undefined' ? window.PUBLIC_URL : null) || 'http://localhost:8086/',
   apiBaseUrl: (typeof window !== 'undefined' ? window.API_BASE_URL : null) || 'http://localhost:8000/',
   uiConfig: typeof window !== 'undefined' ? window.UI_CONFIG : null,
+  openIdClientId: typeof window !== 'undefined' ? window.OPENID_CLIENT_ID : null,
```

Comment:
Here we expose the client ID to the client for anybody to see. Is the client id public data??

Reply:
It is indeed. It is included in the “authorize” call made in users browser.

Reply:
Ah yeah. And the client id may indeed be a legible string (like in https://github.com/City-of-Helsinki/kukkuu-ui/blob/develop/.env#L5), as it does not need to be secret.